kv/kvnemesis: TestKVNemesisSingleNode failed

kv/kvnemesis.TestKVNemesisSingleNode [failed](https://tanzanite.cluster.engflow.com/invocations/default/495903be-c454-4e19-9c41-e93e934afd7b?testReportRun=30&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZuZW1lc2lzOmt2bmVtZXNpc190ZXN0) on master @ [f740c076ec0d92972a47cf6633cf065c8c98678f](https://github.com/cockroachdb/cockroach/commits/f740c076ec0d92972a47cf6633cf065c8c98678f):

Fatal error:

```
panic: test timed out after 4m57s
running tests:
	TestKVNemesisSingleNode (4m54s)

```
Stack: 

```
goroutine 85585 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2259 +0x3b9
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestKVNemesisSingleNode
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestKVNemesisSingleNode4120009455
    test_log_scope.go:81: use -show-logs to present logs inline
    kvnemesis_test.go:327: seed: 5131562799898371130
    kvnemesis_test.go:234: kvnemesis logging to /var/lib/engflow/worker/work/1/exec/_tmp/e2608b3e5de5271ebe05e919d261d295/kvnemesis4171750900
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>run=30</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #115694 kv/kvnemesis: TestKVNemesisSingleNode failed [A-kv-transactions C-bug C-test-failure O-robot P-1 T-kv branch-release-23.2]
</p>
</details>
/cc @cockroachdb/kv
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestKVNemesisSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-35451

kv/kvnemesis.TestKVNemesisSingleNode [failed](https://tanzanite.cluster.engflow.com/invocations/default/6cd93485-5b44-4bde-905f-528c2d7b9a3c?testReportRun=18&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZuZW1lc2lzOmt2bmVtZXNpc190ZXN0) on master @ [3c837c2a86165188a26e8629b7e21b7ae0fb56ae](https://github.com/cockroachdb/cockroach/commits/3c837c2a86165188a26e8629b7e21b7ae0fb56ae):

Fatal error:

```
panic: test timed out after 4m57s
running tests:
	TestKVNemesisSingleNode (4m55s)

```
Stack: 

```
goroutine 63935 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2259 +0x3b9
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestKVNemesisSingleNode
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestKVNemesisSingleNode1351152789
    test_log_scope.go:81: use -show-logs to present logs inline
    kvnemesis_test.go:327: seed: 1459129335189392956
    kvnemesis_test.go:234: kvnemesis logging to /var/lib/engflow/worker/work/3/exec/_tmp/fb930ff13537ce55361891aaf7d31bf9/kvnemesis1960930861
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>run=18</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #115694 kv/kvnemesis: TestKVNemesisSingleNode failed [A-kv-transactions C-bug C-test-failure O-robot P-1 T-kv branch-release-23.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestKVNemesisSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

kv/kvnemesis.TestKVNemesisSingleNode [failed](https://tanzanite.cluster.engflow.com/invocations/default/935c8cf0-cd93-4eb6-b190-b9fabef6e33f?testReportRun=16&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZuZW1lc2lzOmt2bmVtZXNpc190ZXN0) on master @ [0baf22a03d5f55e2611701bc723e3e0b713ab051](https://github.com/cockroachdb/cockroach/commits/0baf22a03d5f55e2611701bc723e3e0b713ab051):

Fatal error:

```
panic: test timed out after 4m57s
running tests:
	TestKVNemesisSingleNode (4m55s)

```
Stack: 

```
goroutine 57818 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2259 +0x3b9
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestKVNemesisSingleNode
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestKVNemesisSingleNode2899087022
    test_log_scope.go:81: use -show-logs to present logs inline
    kvnemesis_test.go:328: seed: 1605344677229518599
    kvnemesis_test.go:235: kvnemesis logging to /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/testlogs/pkg/kv/kvnemesis/kvnemesis_test/run_16_of_30/test.outputs/kvnemesis1012630944
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>run=16</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestKVNemesisSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/kv/kvnemesis/BUILD.bazel
+++ b/pkg/kv/kvnemesis/BUILD.bazel
@@ -60,7 +60,7 @@ go_library(
 
 go_test(
     name = "kvnemesis_test",
-    size = "medium",
+    size = "large",
     srcs = [
         "applier_test.go",
         "engine_test.go",

